front/src/modules/trainschedule/components/Timetable/PacedTrain/PacedTrainItem.tsx

```diff
@@ -0,0 +1,141 @@
+import { useState } from 'react';
+
+import { Checkbox } from '@osrd-project/ui-core';
+import { ChevronDown, Clock, Flame, Manchette } from '@osrd-project/ui-icons';
+import cx from 'classnames';
+import dayjs from 'dayjs';
+import { useTranslation } from 'react-i18next';
+
+import { Duration } from 'utils/duration';
+import { ms2min } from 'utils/timeManipulation';
+
+import TimetableItemActions from '../TimetableItemActions';
+import type { TrainScheduleWithDetails } from '../types';
+
+export type PacedTrain = TrainScheduleWithDetails & {
+  paced: {
+    duration: string;
+    step: string;
```

Comment:
It would be better to use `Duration` I think
```suggestion
    duration: Duration;
    step: Duration;
```

Reply:
Done